index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title> Abel mobile setup </title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+    </head>
+    <body>
+      <div class="home-page">
+        <header>
+          <nav id="nav">
+            <a class="logo" href="#">Abel</a> 
+
+            <nav class="hamburger"><img src="image/Union.png" alt="Menu"></nav>
+            <ul class="internal-link">
+               <li><a href="#my_work">Portfolio</a></li>
+               <li><a href="#about_me">About</a></li>
+               <li><a href="#form-section"> Contact </a></li>
+            </ul>
+
+          </nav>
+        </header>
+        <section id="home_presentation">
+           <h4 class="short-presentation">I’m Abel Fikre
+            Glad to see you!</h4>
+            <p>
+                I’m a Software Development Engineer and Cloud Architect! I can help you build a product ,
+                feature or website Look through some of my work and experience and publish it in the Cloud!
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
```

Comment:
- [ ] As I mentioned above :point_up_2: please stick to the Figma design but your background image is not matching with the Figma design and also the font style of the main heading is not matched with Figma design :+1: check below :point_down:  the comparison between your design and Figma design :slightly_smiling_face:  

**Your Design**
![Screenshot from 2023-06-09 12-53-42](https://github.com/abel12-pro/Portfolio_mobileFirst/assets/45226848/d483e71a-2b73-498a-b510-79d865f4c1ff)

**Figma Design**
![Screenshot from 2023-06-09 12-55-10](https://github.com/abel12-pro/Portfolio_mobileFirst/assets/45226848/a2bb37a7-d99a-4711-9c31-ad33fda423c5)